src/plugins/workspace/public/plugin.ts

```diff
@@ -298,23 +314,25 @@ export class WorkspacePlugin
       core.http.basePath.getBasePath()
     );
 
-    if (workspaceId) {
+    const workspaceError = core.workspaces.workspaceError$;
```

Comment:
```suggestion
    const workspaceError$ = core.workspaces.workspaceError$;
```
We'd better suffix with a $ to indicate it is a observable.

Reply:
Done